keanu-docs/getting-started.md

```diff
@@ -61,7 +60,7 @@ in your gradle or maven build file.
 In your project's build.gradle:
 
 ```$groovy
-compile group: 'io.improbable', name: 'keanu', version: '0.0.8'
```

Comment:
Good catch! This is supposed to be done as part of our release process but we seem to be falling down on that one. Ideally, we would automate this somehow. There might be a way to just link through to Maven Central to the latest version.

Reply:
Yeah - I was thinking the same thing.  As discussed, I'll have a look in to the best ways to go about this over the weekend.